app/src/main/java/com/innowisegroup/datetimepicker/FragmentDatePicker.kt

```diff
@@ -62,71 +46,64 @@ class FragmentDatePicker : Fragment() {
         day?.maxValue = localDate?.lengthOfMonth()!!
         day?.value = localDate?.getDay()!!
         day?.setDividerColor(
-                day,
-                ResourcesCompat.getDrawable(resources, R.drawable.number_picker_divider_color, null)
+            day,
+            ResourcesCompat.getDrawable(resources, R.drawable.number_picker_divider_color, null)
         )
         day?.setFormatter { i: Int ->
             String.format(
-                    Locale.getDefault(),
-                    "%02d",
-                    i
+                Locale.getDefault(),
+                "%02d",
+                i
             )
         }
         day?.wrapSelectorWheel = wrapSelectionWheel
-        day?.setOnValueChangedListener { _: NumberPicker?, _: Int, newVal: Int ->
+        day?.setOnValueChangedListener { _, _, newVal: Int ->
             refreshDateValue(
-                    localDate?.withDayOfMonth(newVal)!!
+                localDate?.withDayOfMonth(newVal)!!
             )
         }
```

Comment:
Let's avoid duplications in code

        day?.run {
            minValue = DAY_MIN_VALUE
            maxValue = localDate?.lengthOfMonth()!!
            value = localDate?.getDay()!!
            setDividerColor(
                day,
                ResourcesCompat.getDrawable(resources, R.drawable.number_picker_divider_color, null)
            )
            setFormatter { i: Int ->
                String.format(
                    Locale.getDefault(),
                    "%02d",
                    i
                )
            }
            wrapSelectorWheel = wrapSelectionWheel
            setOnValueChangedListener { _, _, newVal: Int ->
                refreshDateValue(
                    localDate?.withDayOfMonth(newVal)!!
                )
            }
        }